CODE_OF_CONDUCT.md

```diff
@@ -23,18 +14,25 @@ All recipients of reports commit to:
     * **Node-RED:** <team@nodered.org>
     * **Webdriver.io:** [contact TSC reps](https://github.com/webdriverio/webdriverio/blob/HEAD/AUTHORS.md)
     * **Webhint:** <support@webhint.io>
-2. For every other OpenJS Foundation project, please email <report@lists.openjsf.org>. The Cross Project Council (CPC) is responsible for managing these reports.
+    
+    In case of a conflict of interest or if the report is not handled appropriately, you may [escalate the report][escalation] to the [Code of Conduct Enforcement Team (CoCET)][CoCET] by emailing <coc-escalation@lists.openjsf.org> or by contacting any if its members by email.
+2.  **To report any other incident** (for example: in an OpenJS Foundation project not listed above, in a Collab Space, in the Cross Project Council, or during an event), please email the [Code of Conduct Enforcement Team (CoCET)][CoCET] at <report@lists.openjsf.org> or contact any if its members by email.
+
+## Appeal
 
+The victim(s) and offender(s) have thirty (30) days to appeal the decision made in response to an incident report if they believe the process was not properly followed. Appeals are not suspensive.
 
-## Report an issue in a space managed by the foundation
+To appeal a decision, please email the [Code of Conduct Enforcement Team (CoCET)][CoCET] at <coc-appeal@lists.openjsf.org> or contact any if its members by email. TODO: make sure this email exists.
 
-For reporting issues in spaces managed by the OpenJS Foundation, for example, repositories within the OpenJS organization or an live event such as a conferences, email <report@lists.openjsf.org>. The Cross Project Council (CPC) is responsible for managing these reports.
+## Code of Conduct Enforcement Team (CoCET)
 
-## Escalate an issue
+The Code of Conduct Enforcement Team (CoCET) is a foundation-wide team established to manage code of conduct incident reports across the whole foundation (except for [projects who have opted-in][] to manage reports directly), escalation of reports managed by projects, and appeals to decisions made in response to incidents. The CoCET is currently composed of:
```

Comment:
where do/will we document who selects this team and related info?

Reply:
I was just typing in the same comment. It needs to be documented how the team members are selected.

Reply:
it probably shouldn't be in the CoC directly - just somewhere linked from here :-)

Reply:
Yeah, I'm drafting this and figuring out the order of stuff as I go. It'll go in COC_POLICY and maybe a little in COC_PROCESS too. This is very drafty right now, tbh.

Reply:
That said, I do think that the members of the CoCET team should be listed in the COC (so reporters can reach out directly if they prefer) and that a line to explain what the CoCET is and referencing its full definition is thus warranted.

Reply:
Totally agree; list current members inline, but link to full context/process/etc elsewhere.

Reply:
I agree they should be listed, with a link to doc on how they are selected and approved.